packages/ui-components/style/windowedlist.css

```diff
@@ -4,17 +4,58 @@
  */
 
 .jp-WindowedPanel-outer {
+  height: 100%;
```

Comment:
This breaks position of notebook footer

| Before | After |
|--------|--------|
| ![image](https://github.com/jupyterlab/jupyterlab/assets/5832902/3fb795a8-f5d2-46e7-a7a1-15d7b44e93da) | ![image](https://github.com/jupyterlab/jupyterlab/assets/5832902/7634fee0-cf16-40d4-b632-e7f9fc732cf9) |

Reply:
It looks like the solution is to move `jp-Notebook-footer` inside `jp-WindowedPanel-outer`

Reply:
Thank you! Fixed.